### ISSUE TITLE
Refactor: Deprecate threads-client in favor of the new threads-database

### ISSUE THREAD

**Issue by carsonfarmer** [textileio/js-threads#419]
Part of the "great refactor" (#414). This step depends on #418 #417 #416 and #415 at the very least.

**Comment by carsonfarmer** [comment]
Closing this as "won't fix" because we want to keep client around for low-level access to threaddb instances.
